mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/contact.jsp

```diff
@@ -0,0 +1,82 @@
+<%@ page import="com.liferay.todo.FormController" %>
+<%@ page import="com.liferay.todo.TodoPortlet" %><%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+
+<%@ include file="/init.jsp" %>
+
+<div class="list-container">
+
+	<%@ include file="/header.jspf" %>
+
+	<div class="form-container">
+		<h3>
+			<liferay-ui:message key="contact-us" />
+
+			<i class="icon-envelope"></i>
+		</h3>
+
+		<div id="contact-form">
+			<portlet:actionURL name="formAction" var="formActionURL">
+
+				<portlet:param name="mvcPath" value= "/contact.jsp"></portlet:param>
```

Comment:
You don't need mvcPath param:
- By default, the page will refresh. (redirect page is the same)
- You are explicitly setting it in TodoPortlet:55
- For redirection, you can use ```redirect``` parameter, see https://github.com/liferay/liferay-plugins-ee/blob/ee-6.2.x/portlets/osb-lcs-portlet/docroot/account/edit_notifications_settings.jsp and  

Also, spacing is off.